### PR TITLE
[zh] Update reference\glossary (11)

### PR DESCRIPTION
**Pull request by ydFu**
Update about `reference/glossary`,this chapter is often used, so make adjustments,and ensure all these files were still in sync with English upstream now.

the following
```
content/zh-cn/docs/reference/glossary/device-plugin.md
content/zh-cn/docs/reference/glossary/disruption.md
content/zh-cn/docs/reference/glossary/docker.md
content/zh-cn/docs/reference/glossary/dockershim.md
```

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh-cn/docs/reference/glossary/device-plugin.md [hunk 1]
```diff
@@ -23,16 +21,20 @@ aka:
...
 <!-- 
  Device plugins run on worker
 {{< glossary_tooltip term_id="node" text="Nodes">}} and provide
 {{< glossary_tooltip term_id="pod" text="Pods ">}} with access to resources,
 such as local hardware, that require vendor-specific initialization or setup
 steps.
 -->
-设备插件工作在节点主机上，给 {{< glossary_tooltip term_id="pod" text="Pods ">}} 提供访问资源的权限，比如特定厂商初始化或者安装的本地硬件。
+设备插件在节点主机上运行
+{{<glossary_tooltip term_id="node" text="节点">}}并提供
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/device-plugin.md:34]:
```suggestion
设备插件在工作{{<glossary_tooltip term_id="node" text="节点">}}上运行并为
```

File: content/zh-cn/docs/reference/glossary/device-plugin.md [hunk 2]
```diff
@@ -23,16 +21,20 @@ aka:
...
+{{<glossary_tooltip term_id="pod" text="Pod">}} 可以访问资源，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/device-plugin.md:35]:
```suggestion
{{<glossary_tooltip term_id="pod" text="Pod">}} 提供访问资源的能力，
```

File: content/zh-cn/docs/reference/glossary/device-plugin.md [hunk 3]
```diff
@@ -23,16 +21,20 @@ aka:
...
+例如本地硬件，需要特定于供应商的初始化或安装步骤。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/device-plugin.md:36]:
```suggestion
例如本地硬件这类资源需要特定于供应商的初始化或安装步骤。
```

File: content/zh-cn/docs/reference/glossary/device-plugin.md [hunk 4]
```diff
@@ -42,11 +44,16 @@ Pods can access hardware features that relate to the Node where that Pod is runn
 You can deploy a device plugin as a {{< glossary_tooltip term_id="daemonset" >}},
 or install the device plugin software directly on each target Node.
 -->
-设备插件将资源告知 {{< glossary_tooltip term_id="kubelet" text="kubelet" >}} ，以便相关节点上运行的工作负载Pod可以访问硬件功能。
+设备插件向
+{{<glossary_tooltip term_id="kubelet" text="kubelet" >}}，这样负载资源
+Pod 可以访问与运行该 Pod 的节点相关的硬件功能。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/device-plugin.md:49]:
```suggestion
设备插件向 {{<glossary_tooltip term_id="kubelet" text="kubelet" >}} 公布资源，以便工作负载
Pod 访问 Pod 运行所在节点上的硬件功能特性。
```

File: content/zh-cn/docs/reference/glossary/disruption.md [hunk 5]
```diff
@@ -23,13 +19,16 @@ short_description: >
...
+<!--
  Disruptions are events that lead to one or more
 {{< glossary_tooltip term_id="pod" text="Pods" >}} going out of service.
 A disruption has consequences for workload resources, such as
 {{< glossary_tooltip term_id="deployment" >}}, that rely on the affected
 Pods.
- -->
+-->
+干扰（Disruption）是指导致一个或者多个 {{< glossary_tooltip term_id="pod" text="Pod" >}} 服务停止的事件。
+干扰会影响工作负载资源，比如 {{< glossary_tooltip term_id="deployment" >}} 这种依赖于受影响 Pod 的资源。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/disruption.md:31]:
```suggestion
干扰会影响依赖于受影响的 Pod 的资源，例如 {{< glossary_tooltip term_id="deployment" >}}。
```

File: content/zh-cn/docs/reference/glossary/docker.md [hunk 6]
```diff
@@ -23,20 +21,17 @@ short_description: >
...
 - fundamental
----
 -->
 
 <!--
 Docker (specifically, Docker Engine) is a software technology providing operating-system-level virtualization also known as {{< glossary_tooltip text="containers" term_id="container" >}}.
 -->
-
- Docker（这里特指 Docker 引擎） 是一种可以提供操作系统级别虚拟化（也称作{{< glossary_tooltip text="容器" term_id="container" >}}）的软件技术。
+Docker（这里特指 Docker 引擎（Docker Engine））是一种可以提供操作系统级别虚拟化（也称作{{< glossary_tooltip text="容器" term_id="container" >}}）的软件技术。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/docker.md:29]:
```suggestion
Docker（这里特指 Docker Engine）是一种可以提供操作系统级别虚拟化
（也称作{{< glossary_tooltip text="容器" term_id="container" >}}）的软件技术。
```

File: content/zh-cn/docs/reference/glossary/dockershim.md [hunk 7]
```diff
@@ -4,13 +4,12 @@ id: dockershim
 date: 2022-04-15
 full_link: /zh-cn/dockershim
 short_description: >
-   dockershim 是 Kubernetes v1.23 及之前版本中的一个组件，Kubernetes 系统组件通过它与 Docker Engine 通信。
+   dockershim 是 Kubernetes v1.23 及之前版本中的一个组件，Kubernetes 系统组件通过它与 Docker 引擎（Docker Engine）通信。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/dockershim.md:7]: 部分翻译 Docker Engine 是不必要的

File: content/zh-cn/docs/reference/glossary/dockershim.md [hunk 8]
```diff
@@ -28,11 +27,11 @@ tags:
 The dockershim is a component of Kubernetes version 1.23 and earlier. It allows the kubelet
 to communicate with {{< glossary_tooltip text="Docker Engine" term_id="docker" >}}.
 -->
-
 dockershim 是 Kubernetes v1.23 及之前版本中的一个组件。
-Kubernetes 系统组件通过它与 {{< glossary_tooltip text="Docker Engine" term_id="docker" >}} 通信。
+Kubernetes 系统组件通过它与 {{< glossary_tooltip text="Docker 引擎（Docker Engine）" term_id="docker" >}}通信。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/dockershim.md:31]: 同上。

**Review by Sea-n — COMMENTED** [review]
/lgtm

**Pull request merged**